### DISCUSSION TITLE
Agent visualization key frame model

### DISCUSSION THREAD

**Discussion by daniele-baccega** [FLAMEGPU/FLAMEGPU2#1106] [Q&A] [answered]
Hello,
is it possible to assign a different key frame model based on an agent variable in a way similar to how you do with colors using a cmap?

Thanks!

**Comment by Robadob** [comment, marked as answer]
Not currently it's not something we had considered, but that's an interesting idea.

I've created an issue to track it https://github.com/FLAMEGPU/FLAMEGPU2-visualiser/issues/130

But I don't think it's a feature we'll be able to add support for in the immediate future (priorities/time constraints). PRs are always welcome if adding it is within your skillset though.
